metricflow/cli/main.py

```diff
@@ -58,7 +58,7 @@
 
 
 @click.group()
-@click.option("-v", "--verbose", is_flag=True)
```

Comment:
Didn't realize `-V` was commonly used - got any examples?

Reply:
wait woops, I for some reason thought this was version command. reverting.